db/seeds.rb

```diff
@@ -329,8 +329,15 @@ def setup_hmis_admin_access
   # Create HMIS Data Source
```

Comment:
We should probably revisit this.  In production, an installation may support multiple HMIS, so we probably don't want to always create a new HMIS Data Source when it's enabled.  Maybe just `return if Rails.env.production?`

Reply:
Yeah, I agree. I split it out like so:
* Test/Dev: set up data source, role, acl (if missing)
* Test/Dev/Production: if there are any HMIS datasources, reload the Form Definitions and Service types

Reply:
So for any production deploy, we will need to set up the data source and initial role manually. I think that's what we want anyway, feels safer.